### ISSUE TITLE
VS Code will go into "background" mode if the window looses focus while the cursor is within the popup terminal but will not resume focus in the editor once you switch back. 

### ISSUE THREAD

**Issue by viertaxa** [Microsoft/vscode#11651]
- VSCode Version: 1.4.0
- OS Version: Win 7 x64

Steps to Reproduce:

**Please see comment below for update**

1. Have file(s) open in the editor
2. Open the popup terminal (ctl+`)
3. VS Code looses focus (IE you look something up in a browser, etc)
4. Move back to VS Code. 
5. Sections within the "editor" area are dimmed as if the window still did not have focus.
6. Menu bar works fine, but you can't interact with the editor, git tool, extension manager, etc.
7. If you pick something that opens within the editor area (ie Debug Console) you can see that open, but you still can't interact with the editor. 
8. Restart VS code to continue working.

**Comment by viertaxa** [comment]
Update:

I found the actual cause. I was switching between VS Code,Windows explorer, and Firefox while working on plugin development. After a build, I would open up my explorer window and drag and drop my extension file into TFS. VS Code was maximized, but explorer was not. When dragging from explorer to FF, the cursor briefly passes over the edge of the VS code window.  I was able to replicate that under certain circumstances VS Code will go into "accept a dropped file" mode and not exit that mode after the cursor leaves the window. It's a little finicky to replicate, but I was able to do so by quickly dragging the icon from explorer to the next monitor, passing over a portion of VS Code in the process. Dragging another file over the window briefly will reset the window as this bug is intermittent.
